.gitignore

```diff
@@ -8,7 +8,6 @@
 *.dll
 *.dylib
 *.dSYM
-*.txt
```

Comment:
Add `data/*.txt` (or similar) back in, to exclude the Unicode data files that are downloaded there.